config/environments/development.rb

```diff
@@ -90,13 +89,13 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.default_url_options = { host: config.app_domain }
   config.action_mailer.smtp_settings = {
-    address: "smtp.gmail.com",
-    port: "587",
-    enable_starttls_auto: true,
-    user_name: '<%= ENV["DEVELOPMENT_EMAIL_USERNAME"] %>',
-    password: '<%= ENV["DEVELOPMENT_EMAIL_PASSWORD"] %>',
-    authentication: :plain,
-    domain: config.app_domain
+    address: ENV["SMTP_ADDRESS"],
+    port: ENV["SMTP_ADDRESS"],
+    authentication: ENV["SMTP_AUTHENTICATION"],
```

Comment:
should this fallback to `:plain` ?

```suggestion
    authentication: ENV["SMTP_AUTHENTICATION"].presence || :plain,
```

Reply:
I couldn't find any verification if there's a default fallback or if there should be one but since you had a similar thought as I, let's do add a fallback.